### ISSUE TITLE
CSS not working in 4.2.5 using Azure Modules Container

### ISSUE THREAD

**Issue by MattHarrisUltima** [ironmansoftware/powershell-universal#2946]
### Version

4.2.5

### Severity

Visual

### Steps to Reproduce

CSS code for test.css:
```
.bluebackground {
    background-color: blue;
}
```

dashboard code:
```
New-UDApp -Title 'PowerShell Universal Test App' -Stylesheets @('http://localhost:5000/testpage/test.css','http://localhost:5000/IA-CloudContent/CSS/tabler-icons.css') -Content {
    New-UDContainer -classname bluebackground -Children {
        New-UDParagraph -Text "Hello"
    }
}
```

### Expected behavior

```console
The container should have a blue background
```

### Actual behavior

```console
The CSS file is not loaded. During an inspect, the CSS page is there but empty.

In our dev environment, we have no CSS on our App. We have tried using published folders, hard coded url and file path in the -stylesheets parameter.

This works in 4.2.4 but is broken in 4.2.5. We have also noticed that when accessing a published folder link we can not read css files on a published folder path.
```

### Additional Environment data

Docker container image: ironmansoftware/universal:4.2.5-modules

### Visuals

![2023-12-07 15_30_49-Tom O'Brien _ Microsoft Teams classic](https://github.com/ironmansoftware/issues/assets/106333485/e94454e4-58a0-4be1-baf2-1096cff45830)
![2023-12-07 15_29_55-Mozilla Firefox](https://github.com/ironmansoftware/issues/assets/106333485/e0d30775-dcdf-454a-94e1-b6f0da6ab213)
![2023-12-07 15_28_56-Developer Tools — PowerShell Universal Test App — http___localhost_5000_test_Hom](https://github.com/ironmansoftware/issues/assets/106333485/95bc6e85-aa6a-432c-9275-6d817237eb87)
![2023-12-07 15_28_32-](https://github.com/ironmansoftware/issues/assets/106333485/3170ab26-aacb-482d-a349-bac70a53a4c0)

**Comment by MattHarrisUltima** [comment]
I am now starting to have some intermittent success pulling some CSS data.

I appear to be unable to view CSS files in text form in a web browser when navigating to the file on a published folder.

Using CSS stylesheets using the -stylesheets parameter seams to have stopped working when you provide 2 stylesheets to the parameter.

**Comment by adamdriscoll** [comment]
This is related to https://github.com/ironmansoftware/issues/issues/2947

I have verified that this is fixed in 4.2.6. 

```
New-UDApp -Title 'PowerShell Universal' -Content {
    New-UDElement -Id 'main' -Tag 'div' -Attributes @{
        className = "main"
        style = @{
            height = 100
            width = 100
        }
    }

    New-UDElement -Tag 'div' -Attributes @{
        className = "notmain"
        style = @{
            height = 100
            width = 100
        }
    }
} -Stylesheets @(
    '/home/test.css',
    '/home/test2.css'
)
```

![image](https://github.com/ironmansoftware/issues/assets/1256531/c40f2f13-b987-47d2-9067-51160e9d4c6d)

![image](https://github.com/ironmansoftware/issues/assets/1256531/737749b7-ece7-47b5-a4c5-bc490af25cb4)
